docs/src/advanced/custom-backend.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Use your own backend
+mainTitle: Advanced
+layout: main.pug
+category: Advanced
+withHeadings: true
+navWeight: 4
+editable: true
+githubSource: docs/src/advanced/custom-backend.md
+---
+
+> This guide is compatible with Vue InstantSearch ≥ 1.6.0.
+
+Advanced InstantSearch users may have the need to query Algolia’s servers from their backend instead of the frontend, while still being able to reuse InstantSearch widgets. Possible motivations could be for security restrictions, for SEO purposes or to enrich the data sent by the custom server (i.e. fetch Algolia data and data from their own servers). If this sounds appealing to you, feel free to follow this guide. Keep in mind though that we, at Algolia, [recommend doing frontend search](https://www.algolia.com/doc/faq/searching/searching-from-the-front-end-or-the-back-end/#we-definitely-recommend-frontend-search) for performance and high availability reasons.
+
+By the end of this guide, you will have learned how to leverage InstantSearch with your own backend architecture to query Algolia.
+
+## How it’s made possible
+
+InstantSearch is the UI part on top a search client with a state managed by a [JavaScript Helper](https://github.com/algolia/algoliasearch-helper-js). These three layers are composable and can be interchanged to leverage the InstantSearch widgets system with different search clients.
```

Comment:
repo link to search client?

Reply:
You mean this link? https://github.com/algolia/algoliasearch-client-javascript

Reply:
yep

Reply:
I link it [at the next sentence](https://github.com/algolia/vue-instantsearch/pull/459/files#diff-541552327531ce24ddba2428315a6de3R22), which makes more sense IMO.